utilities/xmlrpcpp/test/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ if(TARGET xmlrpcvalue_base64)
   target_link_libraries(xmlrpcvalue_base64 xmlrpcpp)
 endif()
 
+catkin_add_gtest(test_base64 test_base64.cpp)
+target_link_libraries(test_base64 xmlrpcpp)
+set_target_properties(test_base64 PROPERTIES COMPILE_FLAGS -std=c++11)
```

Comment:
Should this be per-target vs. setting it for the whole package with `set_directory_properties`?

Reply:
This is the only test that needs C++11, so I figured the cleanest thing to do was enable it on this target.

Reply:
Gotcha.